guides/writing-addons/intro-tutorial.md

```diff
@@ -290,4 +290,31 @@ Lastly, be sure to provide a few notes about how others can contribute to the pr
 
 ## Advanced addon configuration
 
-<!-- Help wanted! -->
+Addons are configured using the `ember-addon` hash in the `package.json` file.  
+
+
+
+```json
+"ember-addon": {
+  "configPath": "tests/dummy/config",
+  "before": "single-addon",
+  "defaultBlueprint": "blueprint-that-isnt-package-name",
+  "demoURL": "http://example.com/ember-addon/demo.html",
+  "after": [
+    "after-addon-1",
+    "after-addon-2"
+  ]
+}
+```
+By default, the `configPath` property is defined to point to the config directory of the test dummy application.  All other properties are optional.
+
+### before and after
+These properties specify whether your ember-addon must run "before" or "after" any other Ember CLI addons. Both of these properties can take either a string or an array of strings. The string is the name of another Ember CLI addon, as defined in the `package.json` of the other addon.
+
+### defaultBlueprint
+Addons have a default blueprint that will automatically run when the addon is installed.  By convention, Ember will run the blueprint named after the `name` property in `package.json` 
+
+You may specify a different name using `defaultBlueprint`. See the [addon blueprints](http://localhost:4200/release/advanced-use/blueprints/#addonblueprints) for more information on the default blueprint.
```

Comment:
```suggestion
You may specify a different name using `defaultBlueprint`. See the [addon blueprints](../../advanced-use/blueprints/#addonblueprints) for more information on the default blueprint.
```

I think that's the way the link should go, but please test it

Reply:
I tested this and it works on the deploy preview 🎉